packages/dashboard/src/components/DashboardFrame/Navigation.tsx

```diff
@@ -5,27 +5,36 @@ import {
     ListItemIcon,
     Collapse,
     Theme,
-    ListItemProps,
     useMediaQuery,
     experimentalStyled as styled,
     listItemClasses,
 } from '@material-ui/core'
 import { Masks, AccountBalanceWallet, ExpandLess, ExpandMore, Settings } from '@material-ui/icons'
-import { useContext } from 'react'
-import { useRouteMatch } from 'react-router'
+import { forwardRef, useContext, useMemo } from 'react'
+import { useMatch } from 'react-router'
 import { Link, LinkProps } from 'react-router-dom'
-import { Routes } from '../../pages/routes'
+import { RoutePaths } from '../../pages/routes'
 import { DashboardContext } from './context'
 import { MaskNotSquareIcon } from '@dimensiondev/icons'
 import { useDashboardI18N } from '../../locales'
+import type { Omit } from '@material-ui/system'
 
-function ListItemLinkUnStyled({ nested, ...props }: LinkProps & ListItemProps & { nested?: boolean; to: string }) {
-    return <MuiListItem button component={Link} selected={!!useRouteMatch(props.to)} {...props} />
+const ListItemLinkUnStyled = ({ to, ...props }: { to: string; nested?: boolean }) => {
+    const renderLink = useMemo(
+        () =>
+            forwardRef<HTMLAnchorElement | null, Omit<LinkProps, 'to'>>((linkProps, ref) => (
+                <Link to={to} ref={ref} {...linkProps} />
+            )),
+        [to],
```

Comment:
Hmm this part is strange to me... Why not define a new `Link` that internally renames `component` props (from mui) to `as` props (to react-route)?

Reply:
Link does not have `as` props in v6